example/storage/mapping_postgres_eventstore.go

```diff
@@ -81,15 +81,15 @@ func (CreatedMapper) StorageEvent() any {
 }
 
 func (CreatedMapper) ToStorage(event aggregate.Event[*postcard.Postcard]) any {
-	e := event.(*postcard.Created)
-	return &Created{
+	e := event.(postcard.Created)
+	return Created{
 		ID: e.ID,
 	}
 }
 
 func (CreatedMapper) FromStorage(event any) aggregate.Event[*postcard.Postcard] {
 	e := event.(*Created)
```

Comment:
Do we still need pointer here?

Reply:
I guess storage may return pointer but I guess we do not need it.

Reply:
I'll look into it in another PR, as I want to change the Mapper interface a bit.